src/Sensors.cpp

```diff
@@ -19,14 +19,20 @@ void Sensors::loop() {
             if(_onDataCb) _onDataCb();
             dataReady = true;            // only if the main sensor is ready
         }else{
-            if(_onErrorCb)_onErrorCb("-->[W][SENSORS] PM sensor not configured!");
-            dataReady = false;
+                if (isnan(scd30.getCO2())) {       
```

Comment:
The loop() method is the main sensor method for read all sensors, you don't validate it here, maybe this validation only should happen in the own read method.

Reply:
undo to the original